src/frontend/src/components/ProjectDetailCard.tsx

```diff
@@ -0,0 +1,102 @@
+                {wbsPipe(project.wbsNum)} - {project.name}
+              </Typography>
+            </Link>
+          </Grid>
+          <Grid item xs display="flex" justifyContent="flex-end">
+            <FavoriteProjectButton wbsNum={project.wbsNum} projectIsFavorited={projectIsFavorited} />
+          </Grid>
+        </Grid>
+
+        <Grid container sx={{ marginTop: '0.5rem' }}>
+          <Grid item display="flex" xs={4}>
+            <Construction sx={{ mr: 1 }} /> <Typography>{fullNamePipe(project.projectLead)}</Typography>
+          </Grid>
+          <Grid item display="flex" sx={{ marginTop: 0.5 }} xs={4}>
+            <ScheduleIcon sx={{ mr: 1 }} />
+            <Typography>{project.duration} weeks left</Typography>
+          </Grid>
+          <Grid item display="flex" justifyContent="left" sx={{ marginTop: 0.5 }} xs={4}>
+            <TaskIcon sx={{ mr: 1 }} /> <Typography>{`${tasksLeft} task${tasksLeft === 1 ? '' : 's'} left`}</Typography>
+          </Grid>
+          <Grid item display="flex" xs={4}>
+            <Work sx={{ mr: 1 }} /> <Typography>{fullNamePipe(project.projectManager)}</Typography>
+          </Grid>
+          <Grid item display="flex" justifyContent="left" sx={{ marginTop: 0.5 }} xs={4}>
+            <DescriptionIcon sx={{ mr: 1 }} />
+            <Link href={project.gDriveLink} target="_blank" underline="hover">
+              Google Drive
+            </Link>
+          </Grid>
+          <Grid item display="flex" sx={{ marginTop: 0.5 }} xs={4}>
+            <AttachMoneyIcon sx={{ mr: 1 }} /> <Typography>{project.budget}</Typography>
+          </Grid>
+        </Grid>
+
+        <Box sx={{ marginTop: 2 }}>
+          <Typography variant="h6">Active Work Packages:</Typography>
+          {containsWorkPackages ? (
+            project.workPackages
+              .filter((workPackage) => workPackage.status === WbsElementStatus.Active)
+              .sort((a, b) => a.endDate.getTime() - b.endDate.getTime())
+              .map((workPackage) => {
+                return (
+                  <Grid container sx={{ my: 1 }}>
+                    <Grid item xs={6}>
+                      <Link component={RouterLink} to={`/projects/${wbsPipe(workPackage.wbsNum)}`}>
+                        {wbsPipe(workPackage.wbsNum)} - {workPackage.name}
+                      </Link>
+                    </Grid>
+                    <Grid item xs={3} display="flex" justifyContent="left">
+                      <WorkPackageStageChip stage={workPackage.stage} />
+                    </Grid>
+                    <Grid item xs={3} display="flex">
+                      <Box display="flex">
+                        <ScheduleIcon sx={{ mr: 1 }} /> <Typography>{workPackage.duration} weeks left</Typography>
+                      </Box>
+                    </Grid>
+                  </Grid>
+                );
+              })
+          ) : (
+            <Typography>None</Typography>
+          )}
+        </Box>
```

Comment:
nitpick: I'd break out the 3 sections of this card into helper components in this file to make this more readable: CardTitle, ProjectDetails, WorkPackages